book/02-git-basics/sections/viewing-history.asc

```diff
@@ -224,40 +224,40 @@ Those are only some simple output-formatting options to `git log` – there are
-You've seen one such option already – the `-2` option, which show only the last two commits.
-In fact, you can do `-<n>`, where `n` is any integer to show the last `n` commits.
-In reality, you're unlikely to use that often, because Git by default pipes all output through a pager so you see only one page of log output at a time.
+Em adição ás opções de formatação do retorno, `git log` leva um número útil de opções de limitação – que são, opções que lhe permitem mostrar apenas um subconjunto de commits.
+Você já viu essa opção antes – a opção `-2`, a qual mostra apenas os dois últimos commits.
+De fato, você pode usar `-<n>`, onde `n` é qualquer número inteiro para mostra os ultimos `n` commits.
+Na verdade, você não gostará de usar isso frequentemente, pois o Git por padrão enquadra todo o retorno através de uma página então você vê apenas uma página de registros por vez.  
 
-However, the time-limiting options such as `--since` and `--until` are very useful.
-For example, this command gets the list of commits made in the last two weeks:
+Entretanto, as opções de lina do tempo tais como `--since` e `--until` são muito uteis.
+Por exemplo, esse comando retorna a lista de commits feitos nas últimas duas semanas:
 
 [source,console]
 ----
 $ git log --since=2.weeks
 ----
 
-This command works with lots of formats – you can specify a specific date like `"2008-01-15"`, or a relative date such as `"2 years 1 day 3 minutes ago"`.
+Esse comando funciona com um grande número de formatos – você pode determinar uma data específica como `"2008-01-15"`, ou uma data relativa tal como `"2 anos 1 dia 3 minutos atrás"`.
 
-You can also filter the list to commits that match some search criteria.
-The `--author` option allows you to filter on a specific author, and the `--grep` option lets you search for keywords in the commit messages.
-(Note that if you want to specify both author and grep options, you have to add `--all-match` or the command will match commits with either.)
+Você pode também filtrar a lista de commits que combinam com algum critério de busca.
+A opção `--author` permite você filtrar por um autor especifico, e a opção `--grep` permite você procurar por palavras chaves na mensagem do commit.
+(Note que se você quer especificar ambas as opções autor e grep, você tem que adicionar `--all-match` ou o comando irá combinar com qualquer uma delas.)
 
-Another really helpful filter is the `-S` option which takes a string and only shows the commits that introduced a change to the code that added or removed that string.
-For instance, if you wanted to find the last commit that added or removed a reference to a specific function, you could call:
+Uma outra opção de filtro que realmente ajuda muito é `-S` A qual pega um conjunto de caracteres e mostra apenas os commits que introduzem uma mudança no código onde esse conjunto é adicionado ou removido.
```

Comment:
A letra "A" em "A qual pega" deve ser minúscula, visto que não é início de frase.